rules/cloud/azure/azure_ad_account_created_deleted_nonapproved_user.yml

```diff
@@ -0,0 +1,25 @@
+title: Account Created And Deleted By Non Approved Users
+id: c98184ba-4a27-4e10-b7b7-da48e71f4d25
+status: experimental
+description: Detects when accounts are created and deleted by non-approved users. 
+author: Mark Morowczynski '@markmorow', MikeDuddington, '@dudders1'
+date: 2022/08/11
+references:
+    - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-user-accounts#short-lived-accounts
+logsource:
+    product: azure
+    service: auditlogs
+detection:
+    selection:
+        properties.message:
+            - Initiatied By: 'non approved user UPN'
```

Comment:
You can not use a field secarh in a field search

Reply:
If it is the message `- 'Initiatied By: non approved user UPN'`

Reply:
No it's not a search. It's something the human would have to know that some person created an account and all accounts should be created through some other process. For example, let's say all accounts are created from WorkDay service account. Then some User admin named Bob@contoso.com creates accounts on his own. That's when this should alert. Again, these last 8 rules are things we weren't sure how to create an equivalent Sigma rule for based on the logic of the operations guide. We are looking to you and @Neo23x0 help on that. If these ops rules are not things that can be created in Sigma that's fine as well, I can delete the PR request. We just dont have the in-depth knowledge of Sigma rules to create them.

Reply:
I use the logic : I don't know the bad names but the good ones yes